### PR TITLE
fixing updateCollectionSubjets

### PR DESCRIPTION
**Pull request by alefisico**

### CODE REVIEW

File: python/jetToolbox_cff.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 
 def jetToolbox( proc, jetType, jetSequence, outputFile,
-		updateCollection='', updateCollectionSubjets='',
+		updateCollection='', 
+		updateCollectionSubjets='', subjetSize = 0.4,
```
Comment (dmajumder) [inline python/jetToolbox_cff.py]: naming issue: subjetSize should really be the fat jet size for rerunning the subjet b tagging.

File: python/jetToolbox_cff.py [hunk 2]
```diff
@@ -348,22 +349,28 @@ def jetToolbox( proc, jetType, jetSequence, outputFile,
...
-					jetSource = cms.InputTag( updateCollectionSubjets, 'SubJets' ),
-					labelName = jetALGO+'PF'+PUMethod+'SoftDropPacked',
+					jetSource = cms.InputTag( updateCollectionSubjets ),
+					labelName = jetALGO+'PF'+PUMethod+updateSubjetLabel+'Packed',
 					jetCorrections = JEC, 
-		#			btagDiscriminators = bTagDiscriminators,
+					explicitJTA = True,
+					fatJets = cms.InputTag( updateCollection ),
+					rParam = subjetSize,
+					algo = jetALGO,
```
Comment (dmajumder) [inline python/jetToolbox_cff.py]: rParam = subjetSize, should really be the fat jet size

Comment (alefisico) [inline python/jetToolbox_cff.py]: again, I think this subjetSize is not need it at all. I will remove it before merging.

Comment (dmajumder) [inline python/jetToolbox_cff.py]: ok, i see it.

**Pull request merged**